docs/manual/mkdocs.yml

```diff
@@ -91,6 +91,7 @@ nav:
         - navigation node remove: 'cmd/spo/navigation/navigation-node-remove.md'
       - page:
         - page add: 'cmd/spo/page/page-add.md'
+        - page remove: 'cmd/spo/page/page-remove-md'
```

Comment:
I'm afraid you didn't include this page in the PR

Reply:
Done